src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -22,10 +22,12 @@ public class AddDependenciesToProjectJson : Task
         // Additional Dependencies to add to the project.json. May Optionally contain a version.
         // Will Override dependencies present in the project if there is a conflict.
         // AdditionalDependencies required metadata:  Name, Version
-
         [Required]
         public ITaskItem[] AdditionalDependencies { get; set; }
 
+        // Framework section which the additional dependencies apply to.  Empty is the default dependencies section.
```

Comment:
In that example there are no Project References to convert, but your point is taken.  System.Globalization.Calendars appears to be applicable.